lib/hanami/model/configurator.rb

```diff
@@ -25,12 +25,22 @@ class Configurator
       # @api private
       attr_reader :_schema
 
+      # @since x.x.x
+      # @api private
+      attr_reader :callbacks
+
       # @since 0.7.0
       # @api private
       def self.build(&block)
         new.tap { |config| config.instance_eval(&block) }
       end
 
+      # @since x.x.x
+      # @api private
+      def initialize
+        @callbacks = Hash[before: [], after: []]
```

Comment:
Hmm. Not sure `:before` and `:after` are even the right names..._before_ what? _after_ what? 
It's not that this is executed everytime before the connection is used. It's more a hook to _setup_ the connection?